### PR TITLE
Add claim page

### PR DESCRIPTION
**Pull request by 0x-shanks**
## Overvierw
* https://github.com/henkaku-center/omise-interface/issues/14
* https://github.com/henkaku-center/omise-interface/issues/15

## TODO
- [x] Confirmation of operation on testnet

### CODE REVIEW

File: hooks/claim/useClaim.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      toast({
+        title: 'Error',
+        description: handleErrorMessage(error),
+        status: 'error'
+      })
+    },
+    onSuccess() {
+      toast({
+        title: 'Submitted',
+        description: 'Answer was correct.\nYou earn 100 points',
```
Comment (hourglasshoro) [inline hooks/claim/useClaim.ts:19]: [MUST] fix

File: hooks/useUserAttribute.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+type UserAttribute = {
+  point: number
+  claimableToken: number
+  answeredAt: string
+}
+
+export const useUserAttribute = (
+  contract: string,
+  owner: string | undefined
+): { data?: UserAttribute; isError: boolean; error: Error | null } => {
```
Comment (hourglasshoro) [inline hooks/useUserAttribute.ts:14]: I need to debug because I wasn't getting good data locally.

File: hooks/claim/useClaim.ts (hunk 1 above)
Comment (hourglasshoro) [reply hooks/claim/useClaim.ts]: https://github.com/henkaku-center/omise-interface/pull/62/commits/5342f195d938d3044813c3f70a130f47522d8ea4 fixed

File: hooks/useUserAttribute.ts (hunk 2 above)
Comment (hourglasshoro) [reply hooks/useUserAttribute.ts]: https://github.com/henkaku-center/omise-interface/pull/62/commits/5342f195d938d3044813c3f70a130f47522d8ea4 fixed

**Comment by hourglasshoro** [comment]
The values of isBalance, userAttribute do not change immediately(related useContractRead), is there a solution?

File: pages/claim/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  const handleClaim = useCallback(async () => {
+    claim()
+    await refetchClaimableToken()
+  }, [claim, refetchClaimableToken, claimableToken]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  return (
+    <>
+      <Layout>
+        <Heading mt={50}>Claim</Heading>
+        <Flex direction={{ base: 'column', sm: 'row' }}>
```
Comment (SamuraiT) [inline pages/claim/index.tsx:39]: aww, this is nice

**Review by SamuraiT — COMMENTED** [review]
please rebase. I wanna check with latest contract

**Comment by hourglasshoro** [comment]
> please rebase. I wanna check with latest contract

I rebased it, but then I thought about it, and I didn't commit it before this PR https://github.com/henkaku-center/omise-interface/pull/60 was merged, so I guess I didn't have a problem.

**Comment by hourglasshoro** [comment]
I didn't fix balanceOf to not degrate.

**Pull request merged**